include/seahorn/Expr/Smt/Z3.hh

```diff
@@ -482,6 +494,20 @@ public:
     Expr res = z3.toExpr(z3::ast(ctx, Z3_simplify_ex(ctx, ast, m_params)),
                          m_ast_to_expr);
     m_cache.insert({e, res});
+    LOG(
```

Comment:
the LOG messages do not belong here. They are strictly for bvopsem2

Reply:
I have moved them back... can you please let me know why though?

Reply:
the logs are specific for debugging bvopsem. They are of no use for other uses of the simplifier